STEP/CodeGeneration/CodeGenerationVisitor.cs

```diff
@@ -682,9 +689,33 @@ public void Visit(PinDclNode n)
         }
 
         _pinSetup.Append(");\r\n");
-        
+
         // Save variable names as constant declarations and prepend to generated code
         string variableConstant = $"#define {n.Left.Name} {pinVisitor.GetPinCode()}\r\n";
         _stringBuilder.Insert(0, variableConstant);
     }
+
+    private void InitProgramFileHelper()
```

Comment:
Kunne måske bruge en uddybbende kommentar om at det er hugget direkte fra Arduino, så vi ikke har noget ansvar for det hehe

Reply:
good idea

Reply:
Der er lagt en lille kommentar nu 👍